src/main/java/com/google/sps/servlets/GroceryServlet.java

```diff
@@ -125,18 +127,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   // returns items from the query that match the users familyID in the Grocery object    
   private ArrayList<Grocery> fetchGroceries(PreparedQuery familyGrocery, String userEmail) {
     boolean isUserAssigned;
+    Boolean complete;
```

Comment:
nit: `boolean`

Reply:
My answer to this is the same as above

Reply:
Does `boolean` not work? I think it should automatically be able to convert `Boolean` to `boolean`.